atlp-rwanda/script-heroes-bn-backend
# #174797505 eslint for linting

#### What does this PR do?
- Configure Eslint
#### Description of Task to be completed?
- Configure eslint
- Configure prettier
#### How should this be manually tested?
- Clone the repo `git clone https://github.com/atlp-rwanda/script-heroes-bn-backend.git`
- Switch your working directory `cd script-heroes-bn-backend`
- Checkout the branch `git checkout ch-configure-eslint-#174797505`
- In your text editor try to edit some code
> You should see code violations and errors comments
#### Any background context you want to provide?
- In your text editor (Vs code recommended) eslint extension should be installed
-  In your text editor (Vs code recommended) prettier extension should be installed
#### What are the relevant pivotal tracker stories?
- [#174797505](https://www.pivotaltracker.com/n/projects/2465784)
#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/44426710/93126359-87c7c900-f6cc-11ea-828f-860c389b0e93.png)
